### ISSUE TITLE
Discrepency between reference_client and command class documentation

### ISSUE THREAD

**Issue by barrya-acn** [Z-WavePublic/libzwaveip#12]
The reference client adds an extra byte to the frame when requesting exclusion mode, when compared to the SDS13784-1 Z-Wave Network-Protocol Command Class Specification document dated 2017-01-03. The documentation matches the ZW_NODE_REMOVE_FRAME from ZW_classcmd.h.

I cannot tell with certainty which is correct - the client, or the documentation. However I found that not including the extra byte would result in no callback to the registered transmit_done_func_t.

I am using Z/IP gateway v2.59.0.

```
void cmd_remove_node(struct zconnection *zc) {
  int idx = 0;
  char buf[200];

  buf[idx++] = COMMAND_CLASS_NETWORK_MANAGEMENT_INCLUSION;
  buf[idx++] = NODE_REMOVE;
  buf[idx++] = get_unique_seq_no();
  buf[idx++] = 0;
  buf[idx++] = 0x01; /* ADD_NODE_ANY */
  buf[idx++] = 0;    /* Normal power, no NWI */

  zconnection_send_async(zc, buf, idx, 0);
}
```

**Comment by jburon-sigma** [comment]
The documentation is correct. The NODE_REMOVE command should not include the 6th byte.
I have removed the extra byte in PR https://github.com/Z-WavePublic/libzwaveip/pull/13 . 
In my setup, the callback function is still called when omitting that byte.

Please test and let me know if you still have trouble with the callback function.

**Comment by jburon-sigma** [comment]
Closing. Fix has been merged in commit 4797742.
